### PR TITLE
tools: convert man pages from man format to mdoc format

### PR DESCRIPTION
**Pull request by bluetech**
The mdoc is more semantic and consistent.

Signed-off-by: Ran Benita <ran@unusedvar.com>

### CODE REVIEW

**Comment by bluetech** [comment]
@whot I like the mdoc syntax better, it's a bit more modern/semantic (but just as cryptic). WDYT?

File: tools/xkbcli-how-to-type.1 [hunk 1]
```diff
@@ -1,44 +1,41 @@
...
+.Os
+.
+.Sh NAME
+.Nm "xkbcli how\-to\-type"
+.Nd query how to type a given Unicode codepoint
+.
+.Sh SYNOPSIS
+.Nm
+.Op options
+.Ao Ar codepoint Ac
```
Comment (whot) [inline tools/xkbcli-how-to-type.1:12]: not sure it's my terminal font, but whatever this uses is a weird one: `⟩` vs the normal `)`. nope, firefox says it's weird too :)

**Review by whot — APPROVED** [review]
LGTM, thanks. My only comment is that parenthesis weirdness for commands but that's what mdoc does, so *shrug*. I suspect the world will keep on turning, parenthesis oddness or not :)

Thanks, mdoc does indeed look cleaner than roff and seems the better choice here.

File: tools/xkbcli-how-to-type.1 (hunk 1 above)
Comment (bluetech) [reply tools/xkbcli-how-to-type.1]: It does look weird, removed it now.

**Pull request merged**